ext/RMagick/rmutil.c

```diff
@@ -1458,7 +1458,11 @@ rm_progress_monitor(
     VALUE rval;
     VALUE method, offset, span;
 
-    if (ruby_stack_check())
+// Default Ruby minimum stack size
+#define RUBY_VM_THREAD_MACHINE_STACK_SIZE_MIN (  16 * 1024 * sizeof(VALUE)) /*   64 KB or  128 KB */
```

Comment:
Should this be `MAX` instead of `MIN`?

Reply:
At least, Ruby runtime can be able to use minimum stack. 
The Ruby stack size is configurable via `RUBY_THREAD_MACHINE_STACK_SIZE` environment variable.
The user can narrows down the upper limit. So It may not be possible to use MAX value ...

Reply:
Ah, I see.